requirements.txt

```diff
@@ -5,7 +5,6 @@ PyVISA==1.9.1
 sympy==1.3
 gmpy2==2.0.8
 
-certifi==2018.8.24
```

Comment:
@lumip 
certifi cannot be downgraded as it is a distutils package. Why do we fix its version?
https://travis-ci.org/qutech/qupulse/jobs/442140464